StripeFinancialConnections/StripeFinancialConnections/Source/Native/InstitutionPicker/InstitutionTableView.swift

```diff
@@ -218,8 +215,16 @@ final class InstitutionTableView: UIView {
     }
 
     func showLoadingView(_ show: Bool) {
-        loadingView.isHidden = !show
-        bringSubviewToFront(loadingView)  // defensive programming to avoid loadingView being hiddden
+        if show {
+            if loadingView?.superview == nil {
+                let loadingView = InstitutionTableLoadingView()
+                addAndPinSubviewToSafeArea(loadingView)
+                self.loadingView = loadingView
+            }
```

Comment:
if show is true and the superview != nil any need to still run the `bringSubviewToFront`?

Reply:
thank you for the check - the `bringSubviewToFront` previously was defensive programming where _if_ the code changed, the loading view would always go to top

what's unique about code now is that instead of persisting the loading view (ex. it would always be part of view hierarchy), it now gets re-added every single time we show/hide; this _essentially_ acts as the `bringSubviewToFront` because its brought to front each time we change the loading (rather than at `init`). there's certainly still _a chance_ for a 'bug' but its much less and given my current understanding of the code, is impossible atm